include/swift/ABI/ObjectFile.h

```diff
@@ -0,0 +1,113 @@
+//===--- ObjectFile.h - Object File Related Information ------*- C++ -*-===//
+//
+// Object File related data structures.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_ABI_OBJECTFILE_H
+#define SWIFT_ABI_OBJECTFILE_H
+
+#include "llvm/Support/ErrorHandling.h"
+
+namespace swift {
+
+#define MACHO_FIELDMD "__swift5_fieldmd"
```

Comment:
Let's use `constexpr static const StringLiteral` for this. That is what `include/swift/Strings.h` does.